packages/fx-core/resource/package.nls.json

```diff
@@ -255,9 +255,9 @@
   "core.TabSPFxOption.detailNew": "Build UI with SharePoint Framework",
   "core.TabNonSso.label": "Basic Tab",
   "core.TabNonSso.detail": "A simple implementation of a web app that's ready to customize",
-  "core.copilotPlugin.api.noAuth": "None auth",
-  "core.copilotPlugin.api.apiKeyAuth": "API key auth(Bearer token auth)",
-  "core.copilotPlugin.api.oauth": "OAuth(Auth code flow)",
+  "core.copilotPlugin.api.noAuth": "No authentication",
+  "core.copilotPlugin.api.apiKeyAuth": "API Key authentication(Bearer token authentication)",
+  "core.copilotPlugin.api.oauth": "OAuth(Authorization code flow)",
```

Comment:
String 260: Is it authorization code flow or authentication code flow?

Reply:
Hi, @supkasar , the process is known as the authorization code flow in OAuth. For more details, refer to this document: https://learn.microsoft.com/en-us/entra/identity-platform/v2-oauth2-auth-code-flow